services/twitch/twitch-extension-version.service.js

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi'
+import TwitchBase from './twitch-base.js'
+
+const helixSchema = Joi.object({
+  data: Joi.array().required(),
+})
```

Comment:
The schema should ensure that this array contains >0 objects with a version key which is a string.
We don't need to do this in the other twitch badge because we're just counting the length of the array - we don't care what's in it.
I've not tested this, but I think it should probably be something like:

```js
data: Joi.array()
  .items(Joi.object({ version: Joi.string().required() }).required())
  .min(1)
  .required()
```

Docs on input validation: https://github.com/badges/shields/blob/master/doc/input-validation.md